scripts/ENTSOE_capacity_update.py

```diff
@@ -22,13 +22,17 @@
 import yaml
 from utils import ROOT_PATH, run_shell_command
 
-from electricitymap.contrib.config import CONFIG_DIR, ZONES_CONFIG
+from electricitymap.contrib.config import CONFIG_DIR
+from electricitymap.contrib.config.reading import read_zones_config
```

Comment:
Is that function new?

Reply:
Yes and no, it was refractored out of the __init__.py in #5269 but it should be the same. It looks like something is not behaving as expected though but I don't know if that is a new behavior or if it just became evident now.

I'll try and look into this the coming weekend.